.github/workflows/codecov-nightly.yml

```diff
@@ -23,9 +23,9 @@ jobs:
           extra-packages: |
             tracelib=?ignore
             rClr=url::https://github.com/Open-Systems-Pharmacology/rClr/releases/download/v0.9.1/rClr_0.9.1.zip
-            ospsuite.utils=url::https://ci.appveyor.com/api/projects/open-systems-pharmacology-ci/ospsuite-rutils/artifacts/ospsuite.utils.zip?pr=false
-            tlf=url::https://ci.appveyor.com/api/projects/open-systems-pharmacology-ci/tlf-library/artifacts/tlf.zip?pr=false
-            ospsuite=url::https://ci.appveyor.com/api/projects/open-systems-pharmacology-ci/ospsuite-r/artifacts/ospsuite.zip?pr=false
+            ospsuite.utils=url::https://ci.appveyor.com/api/projects/open-systems-pharmacology-ci/ospsuite-rutils/artifacts/ospsuite.utils.zip
+            tlf=url::https://ci.appveyor.com/api/projects/open-systems-pharmacology-ci/tlf-library/artifacts/tlf.zip
+            ospsuite=url::https://ci.appveyor.com/api/projects/open-systems-pharmacology-ci/ospsuite-r/artifacts/ospsuite.zip
```

Comment:
That's not good, the option `pr=false` should be kept, otherwise PRs in progress could be installed, which might be erroneous etc.
I tried the encoded version, but it seems not to work 
e.g. `https://ci.appveyor.com/api/projects/open-systems-pharmacology-ci/ospsuite-rutils/artifacts/ospsuite.utils.zip%3Fpr=false` just entered in the browser fails.

Maybe you can quote the download urls?